### ISSUE TITLE
502 error when I run the page

### ISSUE THREAD

**Issue by thisusernameisavailable** [kodaman2/family-trivia-react#1]
I went to the webpage, and got a 502 error.

**Comment by kodaman2** [comment]
Yeah the project never took off in hacktoberfest unfortunately I got super busy with work and other projects.

Though my plan still is to work on it. I'll try to at least get a working project with dummy data in a few weeks.
